sfge/src/main/java/com/salesforce/config/SfgeConfig.java

```diff
@@ -16,4 +16,16 @@ public interface SfgeConfig {
 
     /** Indicates if a Jorje parse error causes the entire process to stop. */
     boolean shouldIgnoreParseErrors();
+
```

Comment:
New config values to control verbose behavior.
`LogWarningsOnVerbose` would make the output noisier by including all the warnings from SFGE as info events.
`ProgressIncrementsOnVerbose` controls how often we post updates on number of paths analyzed.

Reply:
Do we need these config values anymore? Would removing them simplify the code?